src/org/opensolaris/opengrok/analysis/executables/JarAnalyzer.java

```diff
@@ -60,15 +60,15 @@ public void analyze(Document doc, StreamSource src, Writer xrefOut) throws IOExc
                     xrefOut.append("</b>");
                 }
 
-                doc.add(new TextField("full", ename, Store.NO));
+//                doc.add(new TextField("full", ename, Store.NO)); we cannot do multiple adds from lucene 7, so later call will merge this
```

Comment:
find a better way how to enable multiple .add calls
it might be a lucene bug, that you cannot call it twice ... (since before it worked)
alt. this might be some safeguard on proper stream implementation ?